### PR TITLE
Wire up study actions

### LINKED ISSUE #73: It won't let me add a dissemination without ticking the "fed back to field" checkbox
I should be able to add a dissemination and leave the "fed back to field" checkbox unchecked, and it should work.

screen shot 2016-01-28 at 15 18 43

### PR DESCRIPTION
**Pull request by stevenday**
This PR comprises the changes necessary to wire up the various forms on the
study page, allowing you to create:
- documents
- notes
- publications
- disseminations
- study impacts

As part of this I switched the feature specs to use poltergeist and phantomjs,
so there are a couple of steps to take to make sure you have necessary things
in place. This is taken from the provisioning script, but should work fine on
an existing vm:

```
wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2
tar -xjf phantomjs-1.9.8-linux-x86_64.tar.bz2
sudo ln -s /home/vagrant/phantomjs-1.9.8-linux-x86_64/bin/phantomjs /usr/bin/phantomjs
```

Make sure you run `bundle install` and `rake db:migrate` too.

For #3
Closes #73

### CODE REVIEW

File: app/assets/stylesheets/_action-sheet.scss [hunk 1]
```diff
@@ -104,3 +104,9 @@
     margin-top: 0.5em;
   }
 }
+
+.field_with_errors {
```
Comment (stevenday) [inline app/assets/stylesheets/_action-sheet.scss]: We don't control this selector, it's from Rails, so I'm not going to change it.

File: app/models/document.rb [hunk 2]
```diff
@@ -16,9 +20,23 @@
 
 class Document < ActiveRecord::Base
   include StudyActivityTrackable
+  ALLOWED_CONTENT_TYPES = [
```
Comment (stevenday) [inline app/models/document.rb]: This is done, Hound just doesn't realise because it's a few lines down

**Comment by stevenday** [comment]
I'm ignoring the code climate issues here because the duplication is minimal, and the security issue we already mitigate against by checking the user supplied param against a whitelist.

**Comment by stevenday** [comment]
@davea I think this is ready to review when you are

**Comment by davea** [comment]
Looks good! :+1:

**Pull request merged**